test/test_transforms.py

```diff
@@ -1794,6 +1549,292 @@ def test_randomness(fn, trans, config, p):
     assert p_value > 0.0001
 
 
+@pytest.mark.parametrize('data_type, mode', [
+    ('float', 'L'),
+    ('byte', 'L'),
+    ('short', 'I;16'),
+    ('int', 'I'),
+])
+def test_1_channel_tensor_to_pil_image(data_type, mode):
+    to_tensor = transforms.ToTensor()
+
+    if data_type == 'float':
+        img_data = torch.Tensor(1, 4, 4).uniform_()
+        expected_output = img_data.mul(255).int().float().div(255).numpy()
+    elif data_type == 'byte':
+        img_data = torch.ByteTensor(1, 4, 4).random_(0, 255)
+        expected_output = img_data.float().div(255.0).numpy()
+    elif data_type == 'short':
+        img_data = torch.ShortTensor(1, 4, 4).random_()
+        expected_output = img_data.numpy()
+    elif data_type == 'int':
+        img_data = torch.IntTensor(1, 4, 4).random_()
+        expected_output = img_data.numpy()
+    else:
+        assert False
+
+    for transform in [transforms.ToPILImage(), transforms.ToPILImage(mode=mode)]:
```

Comment:
Here we could parametrize over a boolean ``pass_mode`` as well and remove this for loop:

```py
transform = transforms.ToPILImage(mode=mode) if pass_mode else transforms.ToPILImage()
```

Reply:
Done as suggested!